src/sentry/tasks/auto_resolve_issues.py

```diff
@@ -83,6 +91,8 @@ def auto_resolve_project_issues(project_id, cutoff=None, chunk_size=1000, **kwar
     might_have_more = len(queryset) == chunk_size
 
     for group in queryset:
+        if group.issue_type not in enabled_issue_types:
```

Comment:
Let's try to filter these out in the queryset above on L85

Reply:
I tried adding issue_type__in=enabled_issue_types and was getting an error? so I put it here just to see if it would work. Might need some insight as to why it failed in the filter...

Reply:
Can you tell me more about what failed?

Reply:
Figured it out, since issue_type is a property, I needed to use type instead.